src/SFA.DAS.EmployerIncentives.Web/Models/PaymentStatusModel.cs

```diff
@@ -8,5 +8,14 @@ public class PaymentStatusModel
         public DateTime? PaymentDate { get; set; }
         public bool LearnerMatchNotFound { get; set; }
         public bool HasDataLock { get; set; }
+        public bool ApprenticeNotInLearning { get; set; }
```

Comment:
Would it be better to have this property as "ApprenticeInLearning".  If you wanted to know if the apprentice was in learning then you'd have to apply the double negative !ApprenticeNotInLearning